examples/simple-server.rs

```diff
@@ -103,10 +102,12 @@ async fn init_data_source(db: &Db) -> DataSource {
         .unwrap()
 }
 
-#[async_std::main]
+#[tokio::main]
 async fn main() -> Result<(), Error> {
-    setup_logging();
-    setup_backtrace();
+    // Initialize logging
+    let _ = tracing_subscriber::fmt()
+        .with_env_filter(EnvFilter::from_default_env())
```

Comment:
Hmm so we no longer get all the good stuff with like `RUST_LOG_FORMAT={json,full,pretty}` and ANSI colors? Doesn't really matter for this little example but when it comes time to do the sequencer we will want that stuff

Reply:
I moved this to the `HotShot` crate but I didn't use it for this example because it would need to pull in HS